src/plugins/workspace/config.ts

```diff
@@ -7,6 +7,9 @@ import { schema, TypeOf } from '@osd/config-schema';
 
 export const configSchema = schema.object({
   enabled: schema.boolean({ defaultValue: false }),
+  permission: schema.object({
+    enabled: schema.boolean({ defaultValue: true }),
+  }),
 });
```

Comment:
No so familiar with the schema in configSchema, but it seems the permission can be a undefined so do we need to add additional schema.maybe wrapper out of schema.object?